pkg/noderesourcetopology/pluginhelpers.go

```diff
@@ -80,30 +85,91 @@ func initNodeTopologyInformer(tcfg *apiconfig.NodeResourceTopologyMatchArgs, han
 }
 
 func createNUMANodeList(zones topologyv1alpha2.ZoneList) NUMANodeList {
-	nodes := make(NUMANodeList, 0, len(zones))
-	for _, zone := range zones {
+	// maxNUMAId is 64 just init array with this len
```

Comment:
nit: this comment seems redundant, I'd omit it, the code is clear enough. Or at least rephrase to clarify the intent